macOS/dyld_cache_map_parser.cpp

```diff
@@ -34,6 +34,35 @@ static bool is_page_aligned(uint64_t addr, uint64_t page_size = 0x4000) {
   return !(addr & (page_size-1));
 }
 
+std::string find_dyld_map() {
+  std::string name = "/System/Library/dyld/dyld_shared_cache_arm64e.map";
+
+  // from dyld4::APIs::dyld_shared_cache_find_iterate_text
+  std::string cryptexPrefixes[] = {
+    "/System/Volumes/Preboot/Cryptexes/OS/",
```

Comment:
Some of these end with a slash, and some don't. Looking at the line below
```
std::string cryptex = cryptexPrefixes[i] + "/" + name;
```
A slash already gets added (actually it gets added two times since `name` also starts with a slash, see the other comment), so it should be removed from cryptexPrefixes.